helmclient.go

```diff
@@ -57,6 +57,7 @@ type Config struct {
 	TillerImageName              string
 	TillerImageRegistry          string
 	TillerNamespace              string
+	Upgrading                    bool
```

Comment:
```suggestion
	TillerUpgradeEnabled                    bool
```

Can we rename it to this? Otherwise looks good.